sgm/__init__.py

```diff
@@ -1,5 +1,4 @@
-from .data import StableDataModuleFromConfig
```

Comment:
So this was grabbed from #44? ;-)

Reply:
Right. So could #44 please be merged, so there's only one way to import things from `sgm`?